ui/components/multichain/asset-picker-amount/nft-input/nft-input.tsx

```diff
@@ -22,7 +22,7 @@ export function NFTInput({ integerValue, onChange, className }: NFTInputProps) {
       return;
     }
 
-    onChange(newValue.toPrefixedHexString());
+    onChange(newValue.toPrefixedHexString(), String(newValueAsString));
   };
 
   return (
```

Comment:
Is there a way to make the amount input autofocus? That's the behavior for tokens, but doesn't seem to work for NFTs

Reply:
`autofocus` is super inconsistent so I just added logic to focus on key press if that's okay

Reply:
I see it working when I click Send from the NFTs tab in the overview. But not if I go to the NFTs tab from within Send. Low priority tho, so the latest version lgtm

Reply:
Hmm I see it working for both